Sources/PackageRegistry/RegistryManager.swift

```diff
@@ -0,0 +1,248 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import PackageLoading
+import PackageModel
+
+import TSCBasic
+import TSCUtility
+
+import Foundation
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+
+public enum RegistryError: Error {
+    case invalidOperation
+    case invalidResponse
+    case invalidURL
+    case invalidChecksum(expected: String, actual: String)
+    case invalidPackage
+}
+
+public final class RegistryManager {
+    public static var configuration: URLSessionConfiguration = {
+        var configuration = URLSessionConfiguration.default
+        configuration.httpAdditionalHeaders = [
+            "Accept": "application/vnd.swift.registry.v1",
```

Comment:
Do we need this here or would we rather set this at the request level or do we really need both?

Reply:
I think the `User-Agent` field makes sense as a default. The `Accept` header here is mostly a hedge against programming errors. On reflection, this is more confusing than useful, so I've removed it with 70206b62.